dao/application_dao.go

```diff
@@ -173,3 +173,19 @@ func (a *applicationDaoImpl) Unpause(id int64) error {
 
 	return err
 }
+
+func (a *applicationDaoImpl) IsPaused(id *int64) (bool, error) {
+	var isPaused bool
+
+	err := DB.Debug().
+		Model(&m.Application{}).
+		Select("1").
+		Where("id = ?", id).
+		Where("tenant_id = ?", a.TenantID).
+		Where("applications.paused_at IS NOT NULL").
+		Where("DATE(applications.paused_at) <> '0001-01-01'"). // time.Time's default value
```

Comment:
Where `0001-01-01` came from ? (I have it also in my local DB) Is it something related to go init date values ?

Reply:
Yes, as the `Pause` structure's `paused_at` member is a `time.Time` and not a `*time.Time`, we cannot insert null `paused_at` values in the database using Gorm. I firstly tried to change it on this PR but when I saw it affected multiple files I went the conservative way and added another `Where` clause to make it work.